src/index.js

```diff
@@ -166,15 +231,15 @@ class Dropzone extends React.Component {
     }
   }
 
-  fileAccepted(file) {
+  fileAccepted(file: Object) {
     return accepts(file, this.props.accept);
   }
 
-  fileMatchSize(file) {
+  fileMatchSize(file: Object) {
     return file.size <= this.props.maxSize && file.size >= this.props.minSize;
   }
 
-  allFilesAccepted(files) {
+  allFilesAccepted(files: Array<Object>) {
```

Comment:
Let's define the `DataTransferItemType` based on https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem and reuse it. Also this method return type is missing (`:boolean`)